docs/man/man1/jx-gitops-annotate.1

```diff
@@ -41,9 +41,13 @@ Annotates all kubernetes resources in the given directory tree
     adds Kubernetes resource kinds to exclude. For kind expressions see: 
```

Comment:
man1/*.1?



Is the file name correct here?

Reply:
Yes. Look how man files are named in your system.